### DISCUSSION TITLE
How to reduce waiting time for vehicles

### DISCUSSION THREAD

**Discussion by ts52119** [openTCS/opentcs#67] [Q&A] [answered]
During simulation，Only when vehicle-0008 leaves point-0036 and reaches point-0033, can vehicle-0009 move from point-0037 to point-0036，We want to realize that as long as vehicle-0008 leaves point-0036, 0 vehicle-0009 can be started synchronously, reducing the waiting time of 0009. How to set properties to realize this function.thanks a lot.

![image](https://user-images.githubusercontent.com/121488186/211197417-fdac2c37-6903-4617-bc0b-a4dc620c9df8.png)

**Comment by swltr** [comment, marked as answer]
It is by design that the kernel's traffic management does not send a vehicle to a point that may still be occupied. If you want to change that, you have to modify the kernel's traffic management code. An viable alternative may be to reduce the distance between points.

**Reply by ts52119** [reply]
Thank you so much for your help.We have tried this method(reduce the distance between points),and it reduces the wait time.
